packages/notifi-dapp-example/src/components/HistoryList.tsx

```diff
@@ -56,8 +56,19 @@ export const HistoryList: React.FC<HistoryListProps> = ({
       ) : null}
 
       {isLoading && (
-        <div className="m-auto absolute top-0 left-0 right-0 bottom-0 bg-black bg-opacity-5">
-          <LoadingSpinner />
+        <div className="mt-8 ml-20 bg-white rounded-3xl">
+          <div>
```

Comment:
I would make a loading skeleton  component <LoadingSkeloton/> in case we might want to reuse it sometime. and make it 
```
<LoadingSkeloton/>
<LoadingSkeloton/>
<LoadingSkeloton/>
```

If it makes sense